theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -179,11 +177,11 @@ def fn(x):
                         for node in fg.maker.fgraph.toposort()])
             g_out = fg(data)
 
-            if func is T.max:
+            if True:
```

Comment:
Yes. It make the diff more clear, but I agree, I should make a separate commit that just do the indentation.